DirectProgramming/DPC++/StructuredGrids/1d_HeatTransfer/src/1d_HeatTransfer.cpp

```diff
@@ -34,12 +34,13 @@
 //
 //******************************************************************************
 #include <CL/sycl.hpp>
-#include <dpc_common.hpp>
 #include <fstream>
 #include <iomanip>
 #include <iostream>
+#include "dpc_common.hpp"
```

Comment:
dpc_common.hpp can be found in the dev-utilities include folder. please change your code to reflect:

> // e.g., $ONEAPI_ROOT/dev-utilities/<version>/include/dpc_common.hpp
> #include "dpc_common.hpp"